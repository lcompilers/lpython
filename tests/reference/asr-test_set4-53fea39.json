{
    "basename": "asr-test_set4-53fea39",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_set4.py",
    "infile_hash": "3d78c7ad82aa32c3a4cc5f1a7d44e53b81639194f55672ddc99b4d2d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_set4-53fea39.stdout",
    "stdout_hash": "23bfad9943e2e8e357c231e80e963e3d43273857a5c011ef5739a97e",
    "stderr": "asr-test_set4-53fea39.stderr",
    "stderr_hash": "d9646bd3609c55ff39f57ca435fedc7dabed530caf28caddc9e58a06",
    "returncode": 0
}