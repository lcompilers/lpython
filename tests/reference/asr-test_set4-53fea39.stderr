semantic error: Call to set.add must have exactly one argument
 --> tests/errors/test_set4.py:6:5
  |
6 |     a.add(3, 4)
  |     ^^^^^^^^^^^ 
